ledger-core/v2/testutils/localdebug_tag.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+// +build localdebug
+
+package testutils
+
+const LocalDebug = true
+
+func IsLocalDebug() bool {
```

Comment:
Only one implementation, maybe split: const under tag and method without it?

Reply:
done